spring-boot-samples/pom.xml

```diff
@@ -68,6 +68,7 @@
 		<module>spring-boot-sample-tomcat7-jsp</module>
 		<module>spring-boot-sample-traditional</module>
 		<module>spring-boot-sample-undertow</module>
+		<module>spring-boot-sample-undertow-access-log</module>
```

Comment:
I'll fix that too.
Let me ask you something, is everything ok with Spring Boot AutoConfigure?
I keep receiving two tests error even from spring-projects/spring-boot master branch copy without any change.
```
Failed tests:
  DeviceResolverAutoConfigurationTests.deviceResolverHandlerInterceptorRegistered:98 DeviceResolverHandlerInterceptor was not registered.
  SitePreferenceAutoConfigurationTests.sitePreferenceHandlerInterceptorRegistered:142 SitePreferenceHandlerInterceptor was not registered.
```